src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -418,10 +418,10 @@
     <value>Do you want to close all tabs?</value>
   </data>
   <data name="CloseReadOnlyDialog.CloseButtonText" xml:space="preserve">
-    <value>Close anyway</value>
+    <value>Cancel</value>
   </data>
   <data name="CloseReadOnlyDialog.PrimaryButtonText" xml:space="preserve">
-    <value>Cancel</value>
+    <value>Close anyway</value>
   </data>

```

Comment:
This was going to be one of the cases where we have to rename the whole dialog. *We do not*.

If the read-only dialog was in stable (1.6), and I was not backporting the fix, I would ask you to change the `CloseReadOnlyDialog` name. Here's why:

When the localization build phase runs, it downloads the latest available strings -- WITH NO REGARD FOR VERSION. So, if we change the meaning of an existing string in the preview build (especially to a backwards meaning like this one!), the next _stable_ build will receive the new meaning for all of the translations. . . except for english. The english strings follow the code in the branch, where the translated strings follow the code in _main_ (!)